### DISCUSSION TITLE
Specified Python (at C:\Users\Micro\Models\Doutorado\teste) cannot be found or accessed. Check the path for typos or the permissions of the specified executable

### DISCUSSION THREAD

**Discussion by isidororays** [the-anylogic-company/AnyLogic-Pypeline#69] [General]
Hello everyone, I'm trying to connect to my model in Python. Before updating AnyLogic, it worked normally, I updated the pypiline library, but this error appears: Specified Python (at C:\Users\Micro\Models\Doutorado\teste) cannot be found or accessed. Check the path for typos or the permissions of the specified executable

**Comment by t-wolfeadam** [comment]
Hi sorry, just seeing this now. If you can post screenshots or the model that would be helpful. But you can also try passing a path directly to the python executable.
